### DISCUSSION TITLE
Better Text Highlighting/Selection

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#359] [Feature Requests]
For example, if I want to select a line that is a bullet, I don't want Nota to select the bullet as well...at first. I only want the text. Usually the caret/cursor would be at the end of the line and you press `⌘ + ⇧ + ←` to select the line of text that you want (and most of the time I'm just after the text), then you can cut, delete, or paste over, or whatever.

In Nota, I have to do a little extra thinking in that it makes me take the time to deselect the bullet, header symbol, or checkbox. I'd rather it only select the same text first using the keyboard shortcut I mentioned above, then if I want to select the bullet, checkbox, or Header Symbol, I can press the same shortcut keys again.

If that was confusing, [here's a video](https://share.cleanshot.com/9zNSM0Cf) that shows what I'm talking about and uses Bear to demo the behavior I'd like to see come to Nota.

**Comment by astoilkov** [comment]
Yeah, it all makes sense, thanks for the clear explanation!

Actually, we've implemented this and reverted it. The reason was — we weren't sure about it. It's better for some cases, and worse for others. However, I'm thinking we should do it as others (not only Bear) started doing this and it now feels more natural. We tried it a long time ago and things have changed.

In the meantime, have you tried Expand Selection ⌘E? I use it all the time and in a lot of cases you can use it to do the same. It's not as convenient in this exact case but it's more versatile so at least it's a useful tool in your toolbox.

**Reply by marlonjames71** [reply]
I have used ⌘E and it's great. I think I forget about it more than I'd like. haha And yes I agree that how note apps do the selection now feels natural. That's why there was a disconnect for me with Nota and that tiny detail. Almost every app I use or have used does it this way (Apple Notes, Bear, Craft, Notion...the list goes on).

Thanks for responding. Hopefully this can make it in there. Cheers!
